### DISCUSSION TITLE
Restore Feature on new instance

### DISCUSSION THREAD

**Discussion by Cryptec** [firefly-iii/firefly-iii#4373] [General]
it would be awesome to have an option for downoading all settings / Data (Transactions,  Accounts, Rules) done in firefly III to a backup file, 
and be able to resotre a fresh install on another instance for example with this backup file.

**Comment by JC5** [comment]
It’s deliberately not built in because this is very easy to do by importing and exporting your database. You don’t need a separate Firefly III tool to do this.
